tools/runners/run-test-suite-test262.py

```diff
@@ -132,7 +132,7 @@ def main(args):
 
             if summary_found:
                 print(output, end='')
-                if ('Failed tests' in output) or ('Expected to fail but passed' in output):
+                if ('Failed tests' in output) or ('Failed Tests' in output) or ('Expected to fail but' in output):
```

Comment:
How about we invert the logic, and check if "All tests succeeded" is in the output instead? That way if the output changes later we don't silently ignore potential test fails.

Reply:
Good catch, it works fine with ES5.1 and ES6 version of test262 runner. I'll update the fix.